packages/express/src/storage.ts

```diff
@@ -0,0 +1,25 @@
+import { IncomingMessage } from 'http';
+
+import { Storage, StorageKey } from '@logto/node';
+
+export default class ExpressStorage implements Storage {
+  constructor(private readonly request: IncomingMessage) {}
+
+  async setItem(key: StorageKey, value: string) {
+    this.request.session[key] = value;
+  }
+
+  getItem(key: StorageKey) {
+    const value = this.request.session[key];
+
+    if (value === undefined) {
+      return null;
+    }
+
+    return String(value);
+  }
+
+  removeItem(key: StorageKey) {
+    this.request.session[key] = undefined;
+  }
```

Comment:
Maybe a dumb question: why don't we have `save` function in express storage? Or maybe in the opposite way: why do we need `save` function in Next.js SDK?

Reply:
It is all because of the difference between 2 session plugins.